diffs/metricbeat-7.2.diff

```diff
@@ -0,0 +1,5056 @@
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_metrics_module_literal">
++           <a id="_co_elastic_metricsmodule">
+            </a>
+            <code class="literal">
+             co.elastic.metrics/module
+@@ -196,7 +196,7 @@
+            for the list of supported modules.
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_metrics_hosts_literal">
++           <a id="_co_elastic_metricshosts">
+            </a>
+            <code class="literal">
+             co.elastic.metrics/hosts
+@@ -221,7 +221,7 @@
+            .
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_metrics_metricsets_literal">
++           <a id="_co_elastic_metricsmetricsets">
+            </a>
+            <code class="literal">
+             co.elastic.metrics/metricsets
+@@ -234,7 +234,7 @@
+            List of metricsets to use, comma separated. If no metricsets are provided, default metricsets for the module are used.
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_metrics_period_literal">
++           <a id="_co_elastic_metricsperiod">
+            </a>
+            <code class="literal">
+             co.elastic.metrics/period
+@@ -247,7 +247,7 @@
+            The time interval for metrics retrieval, ie: 10s
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_metrics_timeout_literal">
++           <a id="_co_elastic_metricstimeout">
+            </a>
+            <code class="literal">
+             co.elastic.metrics/timeout
+@@ -260,7 +260,7 @@
+            Metrics retrieval timeout, default: 3s
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_metrics_ssl_literal">
++           <a id="_co_elastic_metricsssl">
+            </a>
+            <code class="literal">
+             co.elastic.metrics/ssl.*
+@@ -279,7 +279,7 @@
+            .
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_logs_raw_literal">
++           <a id="_co_elastic_logsraw">
+            </a>
+            <code class="literal">
+             co.elastic.logs/raw
+@@ -300,10 +300,10 @@
+            overrides every other hint and can be used to create both a single or a list of configurations.
+           </p>
+           <div class="pre_wrapper lang-yaml">
+-           <pre class="programlisting prettyprint lang-yaml">co.elastic.metrics/raw: "[{"enabled\":true,\"metricsets\":[\"default\"],\"module\":\"mockmoduledefaults\",\"period\":\"1m\",\"timeout\":\"3s\"}]"</pre>
+-          </div>
+-          <h5>
+-           <a id="_literal_co_elastic_metrics_processors_literal">
++           <pre class="programlisting prettyprint lang-yaml">co.elastic.metrics/raw: "[{\"enabled\":true,\"metricsets\":[\"default\"],\"module\":\"mockmoduledefaults\",\"period\":\"1m\",\"timeout\":\"3s\"}]"</pre>

```

Comment:
fixed in https://github.com/elastic/beats/commit/c4d849058430222ee583c8aac216d814dffb82c7